win32ss/gdi/ntgdi/line.c

```diff
@@ -531,9 +533,9 @@ NtGdiLineTo(HDC  hDC,
 }
 
 // FIXME: This function is completely broken
```

Comment:
Does this comment match with the actual function behaviour still? Wasn't it fixed yet by this refactoring? :thinking: In case it was, it might make sense to remove the comment.

Reply:
No, it's still very broken.